tests/storage_test.go

```diff
@@ -165,5 +165,29 @@ var _ = Describe("Storage", func() {
 				close(done)
 			}, 240)
 		})
+
+		Context("With ephemeral alpine PVC", func() {
```

Comment:
Move this under `Describe("Starting a VM...)` and change to `with`
I also miss test that 
- touch some file
- restart the VM with ephemeral PVC
- check that the file does not exist

Reply:
The test literally is under Describe("Starting a VM block"), it's just different context. Agree about the test described above, that's on the list.

Reply:
@mpolednik srry, missed it